.ci/install_debian.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+apt-get update
+
+# noninteractive tzdata ( https://stackoverflow.com/questions/44331836/apt-get-install-tzdata-noninteractive )
+export DEBIAN_FRONTEND=noninteractive
+
+# CI specific packages
+apt-get install -y clang valgrind wget unzip
```

Comment:
Yo can probably avoid installing valgrind  if you are not using it for the tests.